src/olympia/stats/templatetags/jinja_helpers.py

```diff
@@ -14,15 +13,11 @@
 def report_menu(context, request, report, obj):
     """Renders navigation for the various statistic reports."""
     if isinstance(obj, Addon):
-        has_privs = False
-        if request.user.is_authenticated and (
-            acl.action_allowed(request, amo.permissions.STATS_VIEW) or
-            obj.has_author(request.user)
-        ):
-            has_privs = True
         tpl = loader.get_template('stats/addon_report_menu.html')
         ctx = {
             'addon': obj,
-            'has_privs': has_privs,
```

Comment:
We no longer need to check for privileges? Or has that logic been moved elsewhere?

Reply:
This wasn't used anymore.